File: languages/django_models/family.py
from .abstract_element_model import AbstractElementModel
from django.db import models

class Family(AbstractElementModel):

    # Identity
    spirit = models.TextField(blank=True, null=True)
    history = models.TextField(blank=True, null=True)
    traditions = models.ManyToManyField("Construct", blank=True, related_name="family_traditions")
    traits = models.ManyToManyField("Trait", blank=True, related_name="family_traits")
    abilities = models.ManyToManyField("Ability", blank=True, related_name="family_abilities")
    languages = models.ManyToManyField("Language", blank=True, related_name="family_languages")
    ancestors = models.ManyToManyField("Character", blank=True, related_name="family_ancestors")

    # World
    reputation = models.TextField(blank=True, null=True)
    estates = models.ManyToManyField("Location", blank=True, related_name="family_estates")
    governs = models.ManyToManyField("Institution", blank=True, related_name="family_governs")
    heirlooms = models.ManyToManyField("Object", blank=True, related_name="family_heirlooms")
    creatures = models.ManyToManyField("Creature", blank=True, related_name="family_creatures")

    def __str__(self):
        return self.name

File: languages/csharp_unity_json_properties/Family.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Family : Element
{
    [JsonProperty("spirit"), TextAttribute("")]
    public string spirit;
    [JsonProperty("history"), TextAttribute("")]
    public string history;
    [JsonProperty("traditions"), ReferenceAttribute(typeof(Construct), true)]
    public string traditions;
    [JsonProperty("traits"), ReferenceAttribute(typeof(Trait), true)]
    public string traits;
    [JsonProperty("abilities"), ReferenceAttribute(typeof(Ability), true)]
    public string abilities;
    [JsonProperty("languages"), ReferenceAttribute(typeof(Language), true)]
    public string languages;
    [JsonProperty("ancestors"), ReferenceAttribute(typeof(Character), true)]
    public string ancestors;
    [JsonProperty("reputation"), TextAttribute("")]
    public string reputation;
    [JsonProperty("estates"), ReferenceAttribute(typeof(Location), true)]
    public string estates;
    [JsonProperty("governs"), ReferenceAttribute(typeof(Institution), true)]
    public string governs;
    [JsonProperty("heirlooms"), ReferenceAttribute(typeof(Object), true)]
    public string heirlooms;
    [JsonProperty("creatures"), ReferenceAttribute(typeof(Creature), true)]
    public string creatures;
}
